data/language/en-GB.txt

```diff
@@ -4321,6 +4321,8 @@ STR_6009    :{SMALLFONT}{BLACK}Select cut height
 STR_6010    :{COMMA2DP32}m
 STR_6011    :{COMMA1DP16}ft
 STR_6012    :{COMMA1DP16}
+STR_6013    :{SMALLFONT}{BLACK}Guests will only pay entrance tickets for rides and services but they won't pay anything to enter the park.
+STR_6014    :{SMALLFONT}{BLACK}Guests will only pay the ticket to enter the park and services but they won't pay anything to enter the rides.
```

Comment:
Suggestion:
Could, and maybe should be shorter to not confuse the player, example:
``{SMALLFONT}{BLACK}Guests will only pay the ticket to enter the park and services{NEWLINE}The rides entries are free.``
and 
``{SMALLFONT}{BLACK}Guests will only pay tickets for rides and services{NEWLINE}They won't pay anything to enter``
I think there is a specific name for entré-fees to park but my English isn't top notch for me so this suggestion is lightly :p